mslib/msui/mpl_qtwidget.py

```diff
@@ -392,6 +392,9 @@ def mouse_move(self, event):
         """
         overwrite mouse_move to print lon/lat instead of x/y coordinates.
         """
+        if isinstance(self.canvas.waypoints_interactor, mpl_pi.LPathInteractor):
```

Comment:
It would be very useful, if the lat/lon/(pressure|flight level) would be displayed in the top right corner just as for the side view. (i.e. lat/lon would be as for the sideview, whereas altitude would be defined by the flight track, not the mouse cursor).